proto/cosmos/gov/v1beta1/gov.proto

```diff
@@ -36,8 +36,9 @@ message TextProposal {
 
   option (gogoproto.equal) = true;
 
-  string title       = 1;
-  string description = 2;
+  string title         = 1;
+  string description   = 2;
+  string tallystrategy = 3;
```

Comment:
Do we need to make all text proposals to have dynamic tally strategy? Or add some configuration for tally strategy for a specific type of proposal is restricted to few listings?